health-services/project-factory/src/server/utils/mailUtil.ts

```
@@ -0,0 +1,87 @@
+
+        const localizationMap = await getLocalizedMessagesHandlerViaLocale(locale, tenantId);
+        logger.info("Step 3: Fetched localization map");
+        
+        const MdmsCriteria: MDMSModels.MDMSv2RequestCriteria = {
+            MdmsCriteria: {
+                tenantId: tenantId,
+                schemaCode: "HCM-ADMIN-CONSOLE.emailTemplate"
+            }
+        };
+
+        logger.info("Step 4: Calling MDMS API with criteria: " + JSON.stringify(MdmsCriteria));
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)